frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -540,3 +552,7 @@ function selectItemUsingCheckbox(item, icon = "table") {
         .click();
     });
 }
+
+function bulkActionsPanel() {
+  return cy.get("[class*=Card]");
+}
```

Comment:
For one-line selector, I'd suggest avoiding extractions like this one. We can use (aliases)[https://docs.cypress.io/api/commands/as] for this purpose.

```javascript
cy.get("[class*=Card]").as("bulkActionsPanel").wtihin(...)

// and then later in the test when you need it again
cy.get("@bulkActionsPanel").doSomethingWithIt();
```

Reply:
Also, we might also start adding data-testids for distinct parts of UI, like @alxnddr did (here)[https://github.com/metabase/metabase/pull/15405#discussion_r604835493].

Reply:
That's awesome we decided to adopt testIDs, will update the selector then, thanks!

Reply:
As long as we use them sparingly and for distinct UI elements, all is good. Definitely much better than using `[class*=...]` we had to use so far.

One other place I'd most definitely use testId are sidebars.